packages/presentation/src/utils/noop.ts

```diff
@@ -0,0 +1,3 @@
+function noopFn() {}
+const noop = noopFn as any;
+export { noop };
```

Comment:
Is this just to avoid repeating code in potential future noops?

Reply:
Yep. I copied this from the existing implementation in `@spinnaker/core` as an example.